apps/src/lab2/projects/utils.ts

```diff
@@ -21,3 +20,8 @@ export function getStandaloneProjectId(): string | undefined {
   const appOptions = getScriptData('appoptions') as PartialAppOptions;
   return appOptions.channel;
 }
+
+export function getProjectType(): string {
```

Comment:
General question - would it be better to get `projectType` from the level, as part of level properties (by adding it to the `summarize_for_lab2` method)? I'm thinking since we always fetch the level anyway, we'd always have the most current project type and not have to worry about the project type in app options being stale. I think we'd also be able to just read the project type directly out of redux in `Lab2.tsx`, rather than having conditional logic to fetch it from the lesson or app options? What do you think?

Reply:
The reason I didn't go with that option was we load the level properties in a subcomponent of `Lab2` (we load it in `ProjectContainer`). We could instead load it here, but it would require refactoring quite a bit to move that load state up, given that `Lab2Wrapper` handles the load state. We need to know the project type in `Lab2 in order to initialize the correct component(s). Do you think it's worth doing that refactoring?

Reply:
Ah okay gotcha. Would `Lab2` still respond to redux updates since it's part of the same provider, even though its child component (ProjectContainer) is the one dispatching the updates? I'm not sure...another option could be to pull out the lab-switching specific stuff into a separate component, since the lab view itself does get rendered inside ProjectContainer. This lab-switching subcomponent could then be the one that listens for updates to projectType and does all the lab rendering logic.
Not sure how big of a refactor it would be, so also cool with doing this in parts, but if we can reduce Lab2's dependency on progressRedux, I think that would be good

Reply:
One small thing to keep in mind is whether things get slower if we have to serialize network requests (e.g. first get level properties and then use its response to decide what to retrieve next) versus making use of existing information (say in `labRedux`, which is essentially working as a kind of cache) so that we can make requests in parallel.

Reply:
Can you clarify what you mean @breville? Right now we will always load level properties, then load project data, it just happens in a lower-level component.

Reply:
True - though in this case we do already block on fetching level properties as it determines whether we use the standalone channel ID or fetch channel ID for the level.

Reply:
Oh, it was more a general comment, but with some of the discussions around using level properties for information, rather than what's already in `labRedux`, it sounded like we might have to wait until we got level properties before we knew what to do next.  I just picked this thread to note this thought. :)